editors/vscode/package.json

```diff
@@ -568,6 +574,26 @@
                 "title": "Show available Typst templates (gallery) for picking up a template to initialize",
                 "category": "Typst"
             },
+            {
+                "command": "tinymist.createLocalPackage",
+                "title": "Create Typst Local Package",
+                "category": "Typst"
+            },
+            {
+                "command": "tinymist.openLocalPackage",
+                "title": "Open Typst Local Package",
+                "category": "Typst"
+            },
+            {
+                "command": "tinymist.pullSyncRepo",
```

Comment:
The name of the command could be bikeshedded. Candidate renaming:
- `pullSyncRepo` -> `pullLocalPackage`.
- `typstSync` -> `syncLocalPackage`.

Additionally, what's different between `pull` and `sync`? Could you describe it for users here?